src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/RemoveAzureRmSqlDatabaseLongTermRetentionBackup.cs

```diff
@@ -0,0 +1,208 @@
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = RemoveBackupDefaultSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 0,
+            HelpMessage = "The name of the Azure SQL Server the database is in.")]
+        [ValidateNotNullOrEmpty]
+        [LocationCompleter("Microsoft.Sql/locations/longTermRetentionServers")]
+        public virtual string LocationName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the server.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = RemoveBackupDefaultSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 1,
+            HelpMessage = "The name of the Azure SQL Server the backup is under.")]
+        [ValidateNotNullOrEmpty]
+        public string ServerName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the database.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = RemoveBackupDefaultSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 2,
+            HelpMessage = "The name of the Azure SQL Database the backup is from.")]
+        [ValidateNotNullOrEmpty]
+        public string DatabaseName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the backup name.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = RemoveBackupDefaultSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 3,
+            HelpMessage = "The name of the backup.")]
+        [ValidateNotNullOrEmpty]
+        public string BackupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the LTR Backup object to remove.
+        /// </summary>
+        [Parameter(ParameterSetName = RemoveBackupByInputObjectSet,
+            Mandatory = true,
+            Position = 0,
+            ValueFromPipeline = true,
+            HelpMessage = "The Database Long Term Retention Backup object to remove.")]
+        [ValidateNotNullOrEmpty]
+        public AzureSqlDatabaseLongTermRetentionBackupModel InputObject { get; set; }
+
+        /// <summary>
+        /// Gets or sets the resource ID of the LTR Backup to remove.
+        /// </summary>
+        [Parameter(ParameterSetName = RemoveBackupByResourceIdSet,
+            Mandatory = true,
+            Position = 0,
+            ValueFromPipeline = true,
+            HelpMessage = "The Resource ID of the Database Long Term Retention Backup to remove.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        /// <summary>
+        /// Defines whether it is ok to skip the requesting of rule removal confirmation
+        /// </summary>
+        [Parameter(HelpMessage = "Skip confirmation message for performing the action")]
+        public SwitchParameter Force { get; set; }
```

Comment:
You only need force if this is a particularly destructive cmdlet that you really want the user to be sure about before running, or if there is an unexpected action that occurs because of it (i.e., a folder deletion will cause all files contained to be deleted).  Do you think this cmdlet qualifies?

Reply:
I think so just because it could be a potential loss of data otherwise with a possibility of it being gone if the database has been dropped. That's my reasoning anyways.